internal/alpini/router/alpini-router-impl/src/main/java/com/linkedin/alpini/router/impl/Router.java

```diff
@@ -107,6 +107,8 @@ default Builder timeoutProcessor(@Nonnull TimeoutProcessor timeoutProcessor) {
 
     Builder connectionLimit(@Nonnull IntSupplier connectionLimit);
 
+    Builder connectionCountRecorder(@Nonnull Consumer<Integer> recorder);
```

Comment:
Since would like to keep both stalling and fail-fast connection handlers, shall we add a config option here to control which one to use?

Reply:
Yes, will do.

Reply:
Added new config.